tools/autograd/derivatives.yaml

```diff
@@ -1975,11 +1975,6 @@
   grad_output: hardtanh_backward(grad, self, min_val, max_val)
   self: zeros_like(grad)
 
-- name: kl_div_backward(Tensor grad_output, Tensor self, Tensor target, int reduction=Mean, *, bool log_target=False) -> Tensor
```

Comment:
In `derivatives.yaml` you'll need to change the derivative of `kl_div` to be `kl_div_backward_aux`, and then rename it to `kl_div_backward` in the next PR.